src/session/session.c

```diff
@@ -935,8 +935,6 @@ main (int argc,
   signal (SIGALRM, SIG_DFL);
   signal (SIGQUIT, SIG_DFL);
   signal (SIGTSTP, SIG_IGN);
```

Comment:
I probably should have dropped this as well for fully cleaning up, but let's wait for the first round of test results.

Reply:
Done.